velox/exec/tests/TableWriteTest.cpp

```diff
@@ -1745,6 +1746,150 @@ TEST_P(AllTableWriterTest, tableWriteOutputCheck) {
 
 // TODO: add partitioned table write update mode tests and more failure tests.
 
+TEST_P(AllTableWriterTest, testColumnStatsForWriter) {
```

Comment:
I'm leaning towards keeping it, because `tableWriteOutputCheck` can test the plan without column stats aggregation, and `testColumnStatsForWriter ` can test plan with column stats aggregation

the stats collection feature can be turned off  by config and session property

Reply:
Yes, let's have a single test with different test options? I think column stats is one test parameter?

Reply:
The test plan would be different, then the test result would be different as well (column aggregation would have more), I tried to move, but then eventually there's not much to reuse and we would have two tests in one test function which doesn't seem straightforward.